### ISSUE TITLE
The parameter update of sincconv

### ISSUE THREAD

**Issue by piplyman** [mravanelli/pytorch-kaldi#224]
SincConv has two parameters that need to update, but when backward, F.conv1d(...) updates all values of self.filters. Can I understand in this way?

**Comment by mravanelli** [comment]
Hi, during gradient computation you have to backpropagate through the
entire computation chain. You thus have to backpropagate through the filter
taps to finally reach the two parameters of each filter. In these two
parameters, you accumulate the gradients coming from all the elements of
the filter. In other words, a little change in one of these parameters
causes a bigger change in all the filter taps.
